packages/front-end/components/Layout/Layout.tsx

```diff
@@ -217,9 +217,13 @@ const navlinks: SidebarLinkProps[] = [
         name: "Import your data",
         href: "/importing",
         path: /^importing/,
-        filter: ({ permissions, permissionsUtils, gb }) =>
+        filter: ({ permissionsUtils, gb }) =>
           permissionsUtils.canViewFeatureModal() &&
-          permissions.check("manageEnvironments" as GlobalPermission) &&
+          // This is a little weird, since this is only checking a user's global role
+          permissionsUtils.canCreateOrUpdateEnvironment({
+            projects: [],
+            id: "FOO",
```

Comment:
Can we use an empty string here or would that not work?  I don't love have "FOO" in our code base for this.  Only checking global role makes sense here since this is the importer, which will create new projects

Reply:
We can - I've updated this here and within `environments.tsx`.